.circleci/config.yml

```diff
@@ -223,6 +225,7 @@ jobs:
     executor:
       name: android/android-machine
       resource-class: large
+      tag: default
```

Comment:
This is from #669 right? Also, seems that this PR might need a rebase since it has some unrelated changes.

Reply:
sorry, I had the wrong base branch, these are gone now